python-sdk/tests/sql/operators/test_run_raw_sql.py

```diff
@@ -154,18 +154,25 @@ def test_handlers():
     Test the handler return desired results
     """
 
+    class val:
+        def __init__(self, val):
+            self.val = [val]
+
+        def values(self) -> list:
+            return self.val
```

Comment:
Deepsource fails with https://deepsource.io/gh/astronomer/astro-sdk/run/43069200-4f92-4d9d-a400-c99cc3e9cb5d/python/PTC-W0052?listindex=0

Reply:
Fixed it